frontend/src/components/pages/Default.tsx

```diff
@@ -37,6 +38,21 @@ const TeamInfoDisplay = () => {
   );
 };
 
+const DesignLibraryExamples = () => {
+  const { teamName, numTerms, members, isActive } = useContext(SampleContext);
```

Comment:
```suggestion
```
can remove since context values aren't being used

Reply:
oops lol missed that, ty!